src/main/resources/db/changelog/db.changelog-RDM-10165.xml

```diff
@@ -0,0 +1,31 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+
+    <changeSet id="rdm-10165" runInTransaction="false" author="kiran.yenigala@hmcts.net">
+
+        <addColumn tableName="case_event">
+            <column
+                    name="proxied_by"
+                    type="varchar(64)"
+                    defaultValue="">
```

Comment:
Does this (and the others in this file) `defaultValue` need to be set to an empty string, when it is nullable and I think `StringUtils.isEmpty()` checks for `null` too? Can we simply remove the default?

Reply:
Yes, We can remove the default value.